configs/common/packages.yaml

```diff
@@ -1,5 +1,10 @@
 # Pin versions and specs when building packages
   packages:
+    all:
```

Comment:
Note. We may want to use MKL on HPCs when using the Intel compilers (i.e. when looking for the best performing code). Maybe it's possible to use single colons here so that we can overwrite it in the site configs, if needed?

Reply:
Yeah, that's smart. Probably want built-in MKL that comes with Intel.

Reply:
I removed the double colons. Depending on how we want to handle it we could list `intel-mkl` or `intel-oneapi-mkl` ahead of openblas and prefer those if they're available. But we can deal with that later.